### ISSUE TITLE
CJS / ESM Issue

### ISSUE THREAD

**Issue by seeARMS** [standard-crypto/farcaster-js#694]
**Describe the bug**
I'm migrating from 5.x to 6.x. I'm getting an exception when importing and using the latest version of this lib:
```sh
file:///Users/colinarms/src/papyrus-monorepo-v2/node_modules/@standard-crypto/farcaster-js-hub-rest/dist/esm/hubRestApiClient.js:1
import axios, { AxiosError } from 'axios';
                ^^^^^^^^^^
SyntaxError: Named export 'AxiosError' not found. The requested module 'axios' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'axios';
const { AxiosError } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:128:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:194:5)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:385:24)
    at async importModuleDynamicallyWrapper (node:internal/vm/module:437:15)

Node.js v17.8.0
```

**To Reproduce**
1. import { NeynarAPIClient } from "@standard-crypto/farcaster-js" in ESM environment
2. run node esbuild.mjs

My esbuild:

```js
import esbuild from "esbuild"
import fs from "fs/promises"
import path from "path"
import { execSync } from "child_process"

const distDir = "./dist"
const copyFilesScript = "yarn copy-files"
const fileExtensions = [".js", ".ts", ".json"]

// Remove the dist folder
execSync(`rm -rf ${distDir}`)

// Traverse the directory and collect all files with specified extensions
async function collectFiles(dir, fileList = []) {
  const files = await fs.readdir(dir)

  for (const file of files) {
    const filePath = path.join(dir, file)
    const stat = await fs.stat(filePath)

    if (stat.isDirectory()) {
      if (!filePath.includes("node_modules")) {
        await collectFiles(filePath, fileList)
      }
    } else {
      if (fileExtensions.includes(path.extname(file))) {
        fileList.push(filePath)
      }
    }
  }

  return fileList
}

console.log("Collecting files...")
const entryPoints = await collectFiles("./")
console.log("Collected " + entryPoints.length + " files")

// Bundle the files using esbuild
try {
  console.log("Building...")
  await esbuild.build({
    entryPoints,
    sourcemap: true,
    loader: { ".json": "copy" },
    format: "esm",
    platform: "node",
    target: "node17",
    outdir: distDir,
    allowOverwrite: true,
    tsconfig: "./tsconfig.json",
  })
  console.log("Copying files...")

  // Copy additional files after bundling is complete
  execSync(copyFilesScript)
} catch (err) {
  console.error(err)
  process.exit(1)
}

```

**Expected behavior**
I can import and use the package successfully in an ESM environment

**Desktop (please complete the following information):**
v 6.0.3

**Comment by gsgalloway** [comment]
Thanks for flagging. Been working on nailing down the CJS + ESM dual build, so this is top of mind.

I've tried to reproduce your error and haven't had success yet. Here's the setup on my side, would you mind taking a quick look and letting me know what's different in your build?
https://github.com/gsgalloway/farcaster-js-test

**Comment by seeARMS** [comment]
I solved this - I was using Axios version 0.2.x in my codebase, but in this library it requires 1.x. I upgraded Axios to the latest version in my codebase and it began working. I guess the mixed Axios deps were getting somehow borked.
